Docker_cm.yml

```diff
@@ -0,0 +1,12 @@
+version: '3.1'
+
+services:
+    postgres:
+        image: postgres:10
+        restart: always
+        environment:
+            POSTGRES_USER: Admin
+            POSTGRES_PASSWORD: qwerty
+            POSTGRES_DB: Car
+        ports:
+           - 8080:5432
```

Comment:
8080 port will be used by tomcat, so since we are not using postgresql locally, let host port be 5432 too.
And rename file as "docker-compose.yml" to use it with simple "docker-compose up" command